flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/JobConfWrapper.java

```diff
@@ -45,6 +46,12 @@ public JobConf conf() {
         return jobConf;
     }
 
+    public String getDefaultPartitionName() {
```

Comment:
Can we add a util method to do this? `JobConfWrapper` is only meant to make JobConf serializable.